common.blocks/i-bem/i-bem.bemhtml

```diff
@@ -425,9 +425,9 @@ def()(function() {
             }
 
             if(isBEM && jsParams) {
-                this._str += ' data-bem="' +
-                    this.attrEscape(JSON.stringify(jsParams)) +
-                    '"';
+                this._str += ' data-bem=\'' +
+                    this.xmlEscape(JSON.stringify(jsParams)).replace(/'/g, '&#39;') +
```

Comment:
can we parametrize `xmlEscape` or add another function somehow to avoid external usage or `replace`? escaping should be encapsulated and it's not good to use raw `replace` outside